examples/Makefile

```diff
@@ -1,11 +1,12 @@
 MPTOP = ../micropython
-CFLAGS = -std=c99 -I. -I$(MPTOP) -DNO_QSTR
```

Comment:
Can you keep this Makefile and create a new one for the InterpreterManager example? Under examples, you can create a directory for e.g., c_examples/ cpp_examples?